react/OrderFormContext.js

```diff
@@ -80,6 +82,7 @@ class ContextProvider extends Component {
     state.orderFormContext.updateOrderForm = this.props.updateOrderForm
     state.orderFormContext.addItem = this.props.addItem
     state.orderFormContext.updateOrderFormProfile = this.props.updateOrderFormProfile
+    state.orderFormContext.updateOrderFormShipping = this.props.updateOrderFormShipping
```

Comment:
I know this is not related to your PR, but could you refact this attribution to the state? You can remove the `defaultState` variable and inline the value in the `state` property, so instead of having an empty function you can reference the props directly.

Reply:
Good suggestion! I'll try doing that

Reply:
So, I managed to remove the defaultState like you asked, but I couldn't really remove those reattributions inside the render. First issue is that the component's methods when called in the attribution to state are undefined, and that breaks the usage of updateToastMessage and updateAndRefetchOrderForm.
Doing that on getDerivedStateFromProps is also not possible because they are instance methods, so they can't be called in static methods like getDerivedStateFromProps.
I think the whole concept of injecting methods into the context via state like that is flawed, but I didn't have time to think of a better solution and do a proper refactor.

Reply:
@hapoza ok, that's better, we were discussing now and we may rewrite this `orderFormContext`, so I can fix this later.